src/key.js

```diff
@@ -11,7 +11,7 @@ var h = syn.helpers,
 		var result;
 
 		try {
-			result = el.selectionStart !== undefined;
+			result = ((el.selectionStart !== undefined) && (el.selectionStart !== null));
```

Comment:
Hopefully pointing out the little things is helpful. I don’t ever want to come across as rude. 😊